en/docs/design/rate-limiting/enforce-throttling-and-resource-access-policies.md

```diff
@@ -1,14 +1,14 @@
-# Enforce Throttling and Resource Access Policies
+# Enforce Rate Limiting and Resource Access Policies
 
-**Throttling** allows you to limit the number of hits to an API during a given period, typically to protect your APIs from security attacks and your backend services from overuse, regulate traffic according to infrastructure limitations and to regulate usage for monetization. For information on different levels of throttling in WSO2 API Manager (WSO2 API-M), see [Throttling tiers](_Setting_Throttling_Limits_) .
+**Rate limiting** allows you to limit the number of hits to an API during a given period, typically to protect your APIs from security attacks and your backend services from overuse, regulate traffic according to infrastructure limitations and to regulate usage for monetization. For information on different levels of rate limiting in WSO2 API Manager (WSO2 API-M), see [rate limiting tiers](_Setting_Throttling_Limits_).
```

Comment:
Path is wrong here
(_Setting_Throttling_Limits_)

Reply:
Fixed.